system/camerad/cameras/ife.h

```diff
@@ -184,25 +184,25 @@ int build_initial_config(uint8_t *dst, const SensorInfo *s, std::vector<uint32_t
   // TODO: remove this
   dst += write_cont(dst, 0xa3c, {
     0x00000003,
-    ((s->frame_width - 1) << 16) | (s->frame_width - 1),
+    ((s->frame_width / s->out_scale - 1) << 16) | (s->frame_width - 1),
```

Comment:
instead of `width / scale` everywhere, we should just have an input size and output size defined